docker-compose-generator/docker-fragments/monero.yml

```diff
@@ -0,0 +1,45 @@
+version: "3"
+
+services:
+  monerod:
+    restart: unless-stopped
+    container_name: btcpayserver_monerod
+    image: kukks/monero:v0.14.1.2
+    deploy:
```

Comment:
remove this, depending on the docker-compose version of the host this will crash.

Reply:
done